### DISCUSSION TITLE
Method 'make' not found in \Filament\Tables\Columns\TextColumn

### DISCUSSION THREAD

**Discussion by mikrosmile** [filamentphp/filament#4225] [Help] [answered]
```
Method 'make' not found in \Filament\Tables\Columns\TextColumn 
```

This error im facing on all resources classes

Tried everything so far, clearing cache, downgrading filament, etc. 

Anyone has experienced it before? 

<img width="823" alt="Снимок экрана 2022-09-27 в 11 20 33" src="https://user-images.githubusercontent.com/2136612/192473089-ae64e80c-3092-4cbf-aee9-5890ac119848.png">

**Comment by danharrin** [comment, marked as answer]
Delete `/vendor` and `composer.lock` and run `composer update`, fresh start. No clue why this would happen.

**Reply by mikrosmile** [reply]
Thanks, it helped. Weird :)
